src/Resources/views/Menu/sonata_menu.html.twig

```diff
@@ -47,4 +47,4 @@
     {% endapply %}
 {% endblock %}
 
-{% block label %}{% if is_link is defined and is_link %}{{ icon|default|raw }}{% endif %}{% if options.allow_safe_labels and item.extra('safe_label', false) %}{{ item.label|raw }}{% else %}{{ item.label|trans({}, translation_domain|default('messages')) }}{% endif %}{% endblock %}
+{% block label %}{% if is_link is defined and is_link %}{{ icon|default|raw }}{% endif %}{% if options.allow_safe_labels and item.extra('safe_label', false) %}{{ item.label|raw }}{% else %}{{ item.label|trans(item.labelAttributes, translation_domain|default('messages')) }}{% endif %}{% endblock %}
```

Comment:
Since the intention of this option has no general purpose, I think it must be renamed with a more specific name, like "labelTranslationPlaceholders" or similar.

Reply:
Thank you, labelAttributes was not the right choice. Please check next commit.